src/css/organisms/filter-list.scss

```diff
@@ -1,5 +1,7 @@
 .filter-list-content {
+  display: -ms-grid;
```

Comment:
Is this not added by autoprefixer?

Reply:
Ah! It looks like autoprefixer [handles grid support slightly differently][1]. We might also need to adjust the browserlist in package.json to include IE.

[1]: https://github.com/postcss/autoprefixer#grid-autoplacement-support-in-ie

Reply:
The autoprefixer didn't work as intended for this scenario, so I set the rules explicitly to get the layout required.